devops/openshift/api-deploy.yml

```diff
@@ -158,6 +158,8 @@ objects:
                     secretKeyRef:
                       key: ${ZEEBE_AUTHORIZATION_SERVER_URL_KEY}
                       name: ${API_SECRET_NAME}
+                - name: APPLICATION_ARCHIVE_DAYS
```

Comment:
The variable should also be declared. Please check queue-consumers for reference.
```
  - name: APPLICATION_ARCHIVE_DAYS
    require: true
```